php-frontend/src/main/java/org/sonar/php/parser/PHPGrammar.java

```diff
@@ -1004,13 +1005,22 @@ public ExpressionTree CONCATENATION_EXPR() {
           UNARY_EXPR()))));
   }
 
+  public ExpressionTree POWER_EXPR() {
```

Comment:
Shouldn't ** have a higher [than unary minus](https://wiki.php.net/rfc/pow-operator)?

Reply:
create SONARPHP-731 and SONARPHP-732